Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+RUN python -m pip install --upgrade pip
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD [ "python", "./similar_dev_search/cli.py" ]
```

Comment:
it should be `entrypoint` - so you can pass arguments when you launch docker
https://habr.com/ru/company/southbridge/blog/329138/